openkfw/TruBudget
# 2083 env vars generator

### Checklist




### Description


### How to test

1. Backup you .env file
2. Temporarily comment `exit 1` on lines 13 and 20 (since this is meant to check if you are on latest main branch, which you are not)
3. Delete your .env file in operation folder 
4. Recreate it using `bash env-vars-generator.sh` script
5. Update `.env` only partially and run script again
6. Script should be able to also add only missing variables when you update `.env.example` file
7. Try duplicates, incorrect lines in `.env`, etc.




Closes #2083
